src/components/StyledProgress/StyledProgress.tsx

```diff
@@ -1,48 +1,57 @@
 import React, { ReactElement } from 'react';
-import { Box, Skeleton, Theme, Typography } from '@mui/material';
+import { Box, Skeleton, Typography } from '@mui/material';
+import { styled } from '@mui/material/styles';
 import { useTranslation } from 'react-i18next';
-import { makeStyles } from 'tss-react/mui';
 import { useLocale } from 'src/hooks/useLocale';
 import { percentageFormat } from '../../lib/intlFormat';
 import MinimalSpacingTooltip from '../Shared/MinimalSpacingTooltip';
 
-const useStyles = makeStyles()((theme: Theme) => ({
-  box: {
-    width: '100%',
-    height: '54px',
-    border: '2px solid #999999',
-    borderRadius: '50px',
-    padding: '2px',
-    position: 'relative',
-    marginBottom: theme.spacing(2),
-    display: 'flex',
-    alignItems: 'center',
-  },
-  progress: {
-    position: 'absolute',
-    left: '2px',
-    height: '46px',
-    minWidth: '46px',
-    maxWidth: '99.6%',
-    borderRadius: '46px',
-    transition: 'width 1s ease-out',
-    width: '0%',
-  },
-  skeleton: {
-    borderRadius: '46px',
-    height: '46px',
-    transform: 'none',
-  },
-  primary: {
-    background: 'linear-gradient(180deg, #FFE67C 0%, #FFCF07 100%)',
-  },
-  secondary: {
-    border: '5px solid #FFCF07',
-  },
-  inline: {
-    display: 'inline',
-  },
-  belowDetails: { position: 'absolute', right: '5px' },
+const ProcessBoxContainer = styled(Box, {
```

Comment:
Did you mean progress?

```suggestion
const ProgressBoxContainer = styled(Box, {
```